src/Microsoft.Azure.WebJobs.Host.Storage/Singleton/BlobLeaseDistributedLockManager.cs

```diff
@@ -2,40 +2,38 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Concurrent;
 using System.Diagnostics;
 using System.Globalization;
+using System.IO;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Extensions.Storage;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
 using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.Extensions.Logging;
-using Microsoft.Azure.Storage;
-using Microsoft.Azure.Storage.Blob;
 
 namespace Microsoft.Azure.WebJobs.Host
 {
     /// <summary>
-    /// Provides a CloudBlob lease-based implementation of the <see cref="IDistributedLockManager"/> service for singleton locking.
-    /// This class can be overridden to control where the container comes from. 
-    /// The default derived implementation is <see cref="CloudBlobContainerDistributedLockManager"/> which is container based. 
-    /// Hosts can provide a derived implementation to leverage the accountName and allow different hosts to share.
+    /// Provides a BlobClient lease-based implementation of the <see cref="IDistributedLockManager"/> service for singleton locking.
     /// </summary>
-    public abstract class StorageBaseDistributedLockManager : IDistributedLockManager
+    internal class BlobLeaseDistributedLockManager : IDistributedLockManager
```

Comment:
So we're removing the public abstract class and ability for people to derive and override GetContainer. That may be OK, we'll just need to verify that the scenarios for which this was added initially are no longer important

Reply:
Consumers of the package can still provide the container. This is done through IAzureStorageProvider's GetWebJobsBlobContainerClient() method.